### PR TITLE
Fix build commands in doc/secure_boot_guide.md

### LINKED ISSUE #154: [Bug] Build commands in doc/secure_boot_guide.md need to be updated
Parts of build commands in doc/secure_boot_guide.md  is too old. We can not use these commands to build secure boot image.
Eg. 
```
cargo xbuild -p td-shim --features "secure-boot" --target x86_64-unknown-uefi --release
```
should be update to
```
cargo xbuild -p td-shim --target x86_64-unknown-uefi --release --features=main,tdx,secure-boot --no-default-features
```

### PR DESCRIPTION
**Pull request by liuw1**
Fix #154 

Signed-off-by: Wei Liu <wei3.liu@intel.com>

### CODE REVIEW

File: doc/secure_boot_guide.md [hunk 1]
```diff
@@ -5,15 +5,15 @@ This guide follows the [Secure Boot Specification](secure_boot.md) for td-shim.
 ## Build td-shim with secure boot feature enabled
 
 ```
-cargo xbuild -p td-shim --features "secure-boot" --target x86_64-unknown-uefi --release
+cargo xbuild -p td-shim --target x86_64-unknown-uefi --release --features=main,tdx,secure-boot --no-default-features
 ```
```
Comment (jiangliu) [inline doc/secure_boot_guide.md:9]: Any reason to specify `--no-default-features`?

Comment (liuw1) [reply doc/secure_boot_guide.md:9]: It is specific for QEMU launch, I will update to remove this flag.

File: doc/secure_boot_guide.md [hunk 2]
```diff
@@ -55,8 +55,8 @@ Using ECDSA NIST P384 as example:
 
 Clear environment varibles CC and AR at first:
 ```
-set CC=
-set AR=
+set CC=clang
```
Comment (jiangliu) [inline doc/secure_boot_guide.md:58]: The comment above says about "clear", the code is actually set, so make them consistent?

Comment (liuw1) [reply doc/secure_boot_guide.md:58]: Yes, I have corrected it. Thank you.

**Review by jyao1 — APPROVED** [review]

**Review by jiangliu — APPROVED** [review]

**Pull request merged**